application-collabora-ui/src/main/resources/Collabora/Code/UI.xml

```diff
@@ -553,25 +550,6 @@ require(['jquery', 'xwiki-l10n!collabora-attachment'],function($, l10n) {
   #end
   #set ($pageEditURL = $xwiki.getDocument($pageEditRef).getURL('get'))
 #end
-#macro (tableButtonTemplate)
-  #set ($logoURL = $xwiki.getDocument('Collabora.Code.UI').getAttachmentURL('collabora-symbol.svg'))
-  #set ($label = $escapetool.xml($services.localization.render("collabora.attachment.title")))
-  #getPageEditURL($pageEditURL)
```

Comment:
(I cannot add a comment where this macro was defined, so I'm adding it here)
 Is `getPageEditURL` still used?

Reply:
It's only needed in UIX, but I moved all UIX macros to a dedicated page and I'm including the reference to avoid duplicated macros.